src/common/ui/setting-text.vue

```diff
@@ -62,6 +63,11 @@ export default {
       return !deepEqual(this.parsedData.value, this.defaultValue || '');
     },
   },
+  watch: {
+    canSave(state) {
+      this.toggleUnloadSentry(state);
```

Comment:
This should be relaxed a bit to allow the sentry when there's an error. Use case: I type something in the json options which is invalid until I properly finish it e.g. `"cursorBlinkRate": 100,`

Reply:
done